### PR TITLE
added andrew dorrell's random file

### PR DESCRIPTION
**Pull request by andrewdorrell**
Give me your worst code trolls!

### CODE REVIEW

**Comment by andrewdorrell** [comment]
This closes issue #18 (at least it will once the file has been populated with some meaningful text)

File: andrewdorrell-random_file.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Some junk goes here
+
+- I guess I'll learn some markdown :-(
```
Comment (sergey179) [inline andrewdorrell-random_file.md]: @andrewdorrell will learn markdown - why sad face?

**Review by sergey179 — COMMENTED** [review]
Sergey added review changes drop down box.

**Review by sergey179 — APPROVED** [review]
Fantastic code change. Approving...!!!

File: andrewdorrell-random_file.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+### Things to do
+- Scenic world is a must https://www.scenicworld.com.au/. If you are up for a walk and some nice scenery on the way, you can walk down and catch the scenic railway up
+
+### Places to eat
+- many great little places up and down the main street
+
+### Bars and hangouts
+- The Carrington (www.thecarrington.com.au/) is a wonderful old hotel that you can stay at or just go hang out at the bar or lounge
+
+
```
Comment (sergey179) [inline andrewdorrell-random_file.md:14]: You need to add the following string here: "----------------"

Comment (sergey179) [inline andrewdorrell-random_file.md:14]: You need to add the following string here: "----------------"

**Review by sergey179 — CHANGES_REQUESTED** [review]
Sergeys request changes summary....

**Review by sergey179 — APPROVED** [review]

**Pull request merged**